package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "bcrypt-nodejs": "^0.0.3",
     "body-parser": "^1.15.2",
+    "class-validator": "^0.7.2",
```

Comment:
No @types/class-validator, is it normal ?

Reply:
Did not find it, can you try ?

Reply:
Ok, class-validator is already a Typescript lib :)